clients/client-java/src/main/java/org/apache/gravitino/client/HTTPClient.java

```diff
@@ -132,18 +132,17 @@ private HTTPClient(
    * @return The response body as a string.
    * @throws RESTException If an error occurs during conversion of the response body to a string.
    */
-  private String extractResponseBodyAsString(CloseableHttpResponse response) {
+  
+private String extractResponseBodyAsString(CloseableHttpResponse response) {
     try {
-      if (response.getEntity() == null) {
-        return null;
-      }
-
-      // EntityUtils.toString returns null when HttpEntity.getContent returns null.
```

Comment:
Can you keep this comment? This deletion has no relation with the PR description I think.